File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
def solution(numbers):
    answer = 0
    
    numbers.sort()
    
    if (numbers[0]*numbers[1]) >= (numbers[len(numbers)-2] *numbers[len(numbers)-1]):
        answer = numbers[0] * numbers[1]
    else:
        answer = numbers[len(numbers)-2] *numbers[len(numbers)-1]
        
    return answer

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.js
function solution(numbers) {
    numbers.sort((a, b) => a - b);
    const mul1 = numbers[0] * numbers[1];
    const mul2 = numbers[numbers.length - 1] * numbers[numbers.length -2];
    
    return mul1 > mul2 ? mul1 : mul2;
}